src/routes/oath.ts

```diff
@@ -189,7 +189,7 @@ router.get('/hubspot', async (req, res) => {
     if(hubspotID) {
         try {
 			const token = generateAccessToken(hubspotID as string);
-			res.status(200).send(`<script>document.cookie="${token}"; window.close();</script>`);
+			res.status(200).send(`<script>document.cookie="token=${token}; SameSite=None; Secure"; window.close();</script>`);
```

Comment:
Do we want to give a more descriptive name to avoid potential conflicts? Integration-token, SaasHubIntegrationToken?

Reply:
Good call!